src/test/java/org/jenkinsci/plugins/gitclient/GitAPITestCase.java

```diff
@@ -3372,6 +3374,27 @@ public void test_getRemoteReferences_withMatchingPattern() throws Exception {
         assertTrue(references.isEmpty());
     }
 
+    /**
+     * Test getRemoteSymbolicReferences with listing all references
+     */
+    public void test_getRemoteSymbolicReferences() throws Exception {
+        assumeThat(hasWorkingGetRemoteSymbolicReferences(), is(true));
```

Comment:
Unfortunately, this file is JUnit 3.  It doesn't act correctly for an assumeThat.  Need to replace it with a simple conditional.  See the [test failures](https://ci.jenkins.io/job/Plugins/job/git-client-plugin/view/change-requests/job/PR-236/lastCompletedBuild/testReport/).